4.3/README.md

```diff
@@ -117,6 +127,89 @@ docker run -it --shm-size=1g \
   --name nominatim \
   mediagis/nominatim:4.3
 ```
+### Configuration Example
+
+A setup example with almost all Flag possibilities including a short explanation:
```

Comment:
Please double check you capitalisation. In English all nouns start with lower case letters.

Reply:
I looked the english grammar up mit gpt and i hope the capitalisation is now better